test/helpers/chai-helpers.ts

```diff
@@ -1,18 +0,0 @@
-import chai from 'chai'
-import { ethers } from 'hardhat'
-
-chai.Assertion.overwriteMethod(
-  'eql',
```

Comment:
why did we remove this?

Reply:
I think this was from the different merges i did